src/object_sync/hello.cpp

```diff
@@ -74,15 +71,20 @@ namespace object_sync {
   // NAN_METHOD is applicable to methods you want to expose to JS world
   NAN_METHOD(HelloObject::hello) {
 
-    // Note: a HandleScope is automatically included inside NAN_METHOD (See the docs at NAN that say:
+    // Note: a HandleScope is automatically included inside NAN_METHOD. See the docs at NAN that say:
     // 'Note that an implicit HandleScope is created for you on JavaScript-accessible methods so you do not need to insert one yourself.'
```

Comment:
Yes, still relevant, in the sense that HandleScopes are required whenever you work with V8 objects. Otherwise you will have a memory leak. We work with V8 objects here, so would appear to be leaking memory. However it is not because `NAN_METHOD` takes care of automatically adding a `HandleScope`.